web/server/entrypoint.go

```diff
@@ -410,6 +439,12 @@ func (s *robotServer) serveWeb(ctx context.Context, cfg *config.Config) (err err
 					}
 				}
 
+				// Update logger registry if log patterns may have changed.
+				if !diff.LogEqual {
+					s.logger.Debug("Detected potential changes to log patterns; updating logger levels")
```

Comment:
Could bump to `Info` but may be overly noisy.

Reply:
I'd be interested if we could debug log all the patterns easily, but ya, debug is probably the right level

Reply:
I agree; given "size" of this PR (number of things I'm trying to cover), will punt to https://viam.atlassian.net/browse/RSDK-8712.